### ISSUE TITLE
Where could I find the project road map?

### ISSUE THREAD

**Issue by ToffeeLu** [spring-projects/spring-data-r2dbc#50]
Hi, I want to ask when will you guys release the first GA version? would it be possible before April?
And also a road map will be helpful.

**Comment by mp911de** [comment]
> Would it be possible before April?

No, it's not possible before April.

TL;DR: There is currently no correlation between topics and dates and no GA date available.

The reasons for that are multiple:

* We need an R2DBC GA release first. This will take some time as the API needs to mature.
* Spring Data R2DBC has a certain set of tickets (see our issue tracker) that need to be implemented
* There are still some components missing: Connection pooling, improved transaction support. We cannot ship without these as it would make little sense. We do not have control over all of these items so we don't know when these things are going to happen.
* The team behind R2DBC is rather small and working occasionally on R2DBC as that's not our primary occupation.

What contributors can do to speed things up:

* Engage in discussion on the mailing list
* Review tickets and pull requests, discuss features and bugs
* Actively contribute to the projects by trying things out and reporting bugs and feature requests
* More actively involvement by contributing code in form of pull requests
